xml/http_servers.xml

```diff
@@ -1907,6 +1907,27 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
+   <fingerprint pattern="^BBC (\d+\.\d+\.\d+); ovbbccb \d+\.\d+\.\d+$">
+      <example>BBC 06.00.083; ovbbccb 06.00.083</example>
+      <example>BBC 06.20.050; ovbbccb 06.20.050</example>
+      <example>BBC 11.10.035; ovbbccb 11.10.035</example>
+      <description>OpenView ovbbccb </description>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.product" value="OpenView ovbbccb"/>
```

Comment:
Based on your examples I would interpret the first as version number as the BBC version and the second as the component version. I think we could accurately fingerprint these components but probably a specific product version.

 OpenView is more of a product family than something we can tie a version to.  For example: OpenView Operations Agent and Openview Performance Agent can both utilize these components which they call  subagents but we cannot determine the which of these agents is installed based on the banners since it is only the subagent versions that are identified. 

I'm not sure how useful subagent details alone would be without the product. I could add a parameter for BBC version to my PR or we could just use your method.